co-simulation/bridge/carla_v0.10_xmlrpc_server.py

```diff
@@ -418,14 +648,68 @@ def update_actor_velocity(self, actor_key: ActorKey, velocity: List[float]) -> b
     def get_all_actors(self) -> Dict[str, Dict[str, Any]]:
         try:
             with self.lock:
```

Comment:
Maybe it'll be clear as I continue reading, but are we multithreading here? I haven't yet seen it. Is the XML RPC server itself handling the threading on a per-RPC basis?